apps/web/tests/components/form/MultiSelect.test.tsx

```diff
@@ -1,48 +1,79 @@
 import React from 'react';
 import { Formik } from 'formik';
 import { render, screen } from '@testing-library/react';
-import { MultiSelect } from '@components';
+import { MultiSelect2 } from '@components';
```

Comment:
I have pretty mixed feelings about having enumerated components. Ideally we would have just a single multi select. If we have more than one, they should have names that clarify their differences.